main/webapp/modules/core/scripts/reconciliation/standard-service-panel.js

```diff
@@ -246,7 +246,11 @@ ReconStandardServicePanel.prototype._wireEvents = function() {
     if (this._service.ui && this._service.ui.access) {
       suggestOptions.access = this._service.ui.access;
     }
-    input.suggestT(suggestOptions);
+    var sanitizedSuggestOptions = {
+      query_param_name: suggestOptions.query_param_name,
+      access: suggestOptions.access,
+    }
+    input.suggestT(sanitizedSuggestOptions);
```

Comment:
Once you have introduced a function to do the sanitization, you could simply invoke it like this:
```suggestion
    input.suggestT(sanitizeSuggestOptions(suggestOptions));
```